diracos/scriptTemplates/bundle_diracos_script_tpl.sh

```diff
@@ -32,19 +32,23 @@ cp -r /tmp/pipDirac/bin/ $DIRACOS/usr/
 
 # Fix the shebang for python
 echo "Fixing the shebang"
-grep -rIl '#!/usr/bin/python' /tmp/diracos | xargs sed -i 's:#!/usr/bin/python:#!/usr/bin/env python:g'
+grep -rIl '#!/usr/bin/python' $DIRACOS | xargs sed -i 's:#!/usr/bin/python:#!/usr/bin/env python:g'
+
+# Fix RPATHs
+set -x
+export CONDA_BASE_TMP=$(mktemp -d)
+(cd "${CONDA_BASE_TMP}" &&
+ curl -LO https://repo.anaconda.com/miniconda/Miniconda3-py37_4.8.2-Linux-x86_64.sh &&
+ bash Miniconda3-py37_4.8.2-Linux-x86_64.sh -b -p "${CONDA_BASE_TMP}/miniconda")
+(source "${CONDA_BASE_TMP}/miniconda/bin/activate" &&
+ conda config --add channels conda-forge --env &&
+ conda install --yes python-magic patchelf py-lief tqdm &&
```

Comment:
Why are you actually using conda to get this dependencies? Why not resolve this the same way as dependencies for diracos?

Reply:
These aren't added into DIRACOS, they're just used to modify the binaries that are already there

Reply:
No, I mean diracos build dependencies are
```
mock rpm-build fedora-packager createrepo python-pip
```
and here you are introducing additional rather implicit requirements for building and they are taken from another source then the initial ones.

Reply:
Yes and it's disgusting. I'm not aware of a good source CentOS 6 compatible RPMs for patchelf/lief and any time spent making them would be better spent making DIRACOS v2.

Also this isn't the only place in DIRACOS that does this:

https://github.com/DIRACGrid/DIRACOS/blob/b84970e1baf72a99e93f76c429d729b5b7258681/diracos/scriptTemplates/fix_pip_requirements_versions_tpl.sh#L48-L52

Reply:
yes, I did this to allow DIRACOS extensions without the need to install mock and the rest.

But I agree with @chrisburr , let's try to focus on v2 maybe ? From the beginning of DIRACOS, I hope its lifespawn is as little as possible :D

Reply:
OK then fine with me. Just one small thing, please execute the integration test on tag: docker and not DIRACOS

Reply:
I'm fairly sure I tried that and couldn't due to the use of docker inside docker. I can try again if you think it should work?

(Though this has nothing to do with this PR)

Reply:
Forgot, ok, then just leave it.